### PR TITLE
1137-show-agent-name

### LINKED ISSUE #1137: Show agent name
- Cards (just under title)
- config, add a column
- health add a tag

### PR DESCRIPTION
**Pull request by mainawycliffe**
Closes #1137

### CODE REVIEW

**Comment by moshloop** [comment]
@mainawycliffe - Can we treat the local or '0000' as effectively null or empty ?

**Comment by mainawycliffe** [comment]
@moshloop sure

File: src/api/services/topology.ts [hunk 1]
```diff
@@ -130,6 +130,26 @@ export const getTopologyComponentsWithLogs = () => {
...
 };
 
+type AgentItem = {
+  id: string;
+  name: string;
+  description: string;
+};
+
+export const getAgentByID = async (id: string) => {
+  const res = await IncidentCommander.get<AgentItem[] | null>(
```
Comment (moshloop) [inline src/api/services/topology.ts:140]: Is this going to be cached ?

Comment (mainawycliffe) [reply src/api/services/topology.ts:140]: Yeah, we are using react query, so it should be cached and only fetched once, even if you send multiple requests at once.

**Pull request merged**